views/update/update-statement-validation-errors.html

```diff
@@ -93,6 +93,9 @@ <h1 class="govuk-heading-xl">{{ title }}</h1>
       }) }}
 
       {% include "includes/continue-button.html" %}
+      {% for statementError in statementErrorList %}
```

Comment:
Should we add a comment here, about why? Or is it clear enough?

Reply:
done